<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="mainPage">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="ItemListGroupBox" name="mainBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Editor</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="boardPage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="BoardListGroupBox" name="boardBox">
            <property name="title">
             <string>Boards</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="itemPage">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="ItemsGroupBox" name="itemBox">
            <property name="title">
             <string>Items</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="flipperPage">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="FlipperListGroupBox" name="flipperBox">
            <property name="title">
             <string>Flipper</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="plungerPage">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="PlungerListGroupBox" name="plungerBox">
            <property name="title">
             <string>Plunger</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="fieldPage">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="FieldListGroupBox" name="fieldBox">
            <property name="title">
             <string>Fields</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="zonePage">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="ZoneListGroupBox" name="zoneBox">
            <property name="title">
             <string>Zones</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="View" name="editorSceneView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="text">
           <string>CLOSE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetBtn">
          <property name="text">
           <string>RESET</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>SAVE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAsBtn">
          <property name="text">
           <string>SAVE AS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadBtn">
          <property name="text">
           <string>LOAD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testPlayBtn">
          <property name="text">
           <string>PLAY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="zoomLabel">
          <property name="text">
           <string>Zoom</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zoomEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rotationLabel">
          <property name="text">
           <string>Rotation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rotationEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Properties</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>view.h</header>
  </customwidget>
  <customwidget>
   <class>ItemListGroupBox</class>
   <extends>QGroupBox</extends>
   <header>itemlistgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoardListGroupBox</class>
   <extends>QGroupBox</extends>
   <header>boardlistgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FieldListGroupBox</class>
   <extends>QGroupBox</extends>
   <header>fieldlistgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemsGroupBox</class>
   <extends>QGroupBox</extends>
   <header>itemsgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FlipperListGroupBox</class>
   <extends>QGroupBox</extends>
   <header>flipperlistgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlungerListGroupBox</class>
   <extends>QGroupBox</extends>
   <header>plungerlistgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZoneListGroupBox</class>
   <extends>QGroupBox</extends>
   <header>zonelistgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
